src/js/structs/Service.js

```diff
@@ -120,6 +120,10 @@ module.exports = class Service extends Item {
     };
   }
 
+  getResidency() {
```

Comment:
Could you add a simple test?
The other props have one too.

Same for getUpdateStrategy().